### PR TITLE
Aci 197 a

### PR DESCRIPTION
**Pull request by evanbret**
Moved code and instructions from https://github.com/wapopartners/Fusion-Training-User-Stories/tree/exercise-01 into the lab.

### CODE REVIEW

**Comment by ghost** [comment]
@evanbret this should be more integrated into the lab instead of an exercise within the lab. So instead of having Acceptance criteria etc, it needs to be turned into step by step instructions merged into the lab instructions

File: README.md [hunk 1]
```diff
@@ -1,40 +1,86 @@
...
+linkText: PropTypes.string.tag({
+      group: 'Call to Action',
+      name: 'Text',
+      description: 'Text to prompt the users to click the link',
+    }),
+    link: PropTypes.url.tag({
+      group: 'Call to Action',
+      name: 'Url',
+    })
+```
```
Comment (ghost) [inline README.md:45]: on this code block and the 2 before it, can we fix the indenting? so the first line has the same indent as it's closing brackets?

File: README.md [hunk 2]
```diff
@@ -1,40 +1,86 @@
...
+Now we will allow the user to directly customize my feature using inline editing for a streamlined experience. Building on the alert bar feature we just created, we provide a clearer and easier way for editors to configure custom fields through inline editing as documented [here](https://redirector.arcpublishing.com/alc/arc-products/pagebuilder/fusion/documentation/recipes/adding-custom-fields.md#inline-editing)
 
-1. Complete the user story from branch `exercise-01`. This includes adding a new section to the layout and a new feature in the `global/alert-bar` directory. After completing the exercise, return here and we will modify it by adding inline editing to a text field in the feature. You can also complete the steps below on the custom field we added to the color block.
+6. Make sure the above ticket is completed. This includes adding a new section to the layout and a new feature in the `global/alert-bar` directory. Now we will modify it by adding inline editing to a text field in the feature. You can also complete the steps below on the custom field we added to the color block.
```
Comment (ghost) [inline README.md]: `Make sure the above ticket is completed` can be removed since its all 1 lab now

**Review by ghost — CHANGES_REQUESTED** [review]
2 small changes, otherwise looks good

File: README.md (hunk 2 above)
Comment (evanbret) [reply README.md]: :+1: missed that phrasing

**Review by ghost — APPROVED** [review]

**Pull request merged**